### ISSUE TITLE
Could not find Help

### ISSUE THREAD

**Issue by jeeznews** [BusinessUSA/BusinessUSA-Usability-Feedback#38]
Initially looked in vain for a way of getting help. When I initially clicked on "support," I got only an option to request an appointment. About five minutes later, I got the option of at least clicking "back." the menu tab appeared. At one point, the Support Center appeared, but when I hit the back button to see how I had gotten there and tried to go forward again, the Support Center no longer was there.

**Comment by MagsStew** [comment]
I agree. I failed the entire section for Help>Support Center. I could not find any of the items. I tried to:
1. Navigate - no luck
2. Search and received over 177K results. Implementing Key Match in search will improve user experience.
3. Site Map - no luck
4. Contact Us  - Ask A Question - Submit Feedback does not have a link to submit.

**Comment by ScatterToo** [comment]
Similar results - no "Help" link anywhere. I had to type in the url
